src/quacc/calculators/espresso/espresso.py

```diff
@@ -326,10 +325,20 @@ def _sanity_checks(self, parameters: dict[str, Any]) -> None:
 
 class Espresso(Espresso_):
     """
-    This is a wrapper around the ASE Espresso calculator that adjusts input_data
-    parameters and allows for the use of presets.
+    This can be viewed as an extension of the ASE Espresso calculator allowing
+    to run calculations not only using the main program `pw.x` but also other modules.
+
+    The full list is available in the [Quacc list of recipes](https://quantum-accelerators.github.io/quacc/user/recipes/recipes_list.html#quantum-espresso).
 
-    Templates are used to set the binary and input/output file names.
+    Instruction for the impatient:
+
+    1. If you are used to ASE, you can send the same parameter to Quacc's jobs `input_data`, `kpts` ...
+    2. In Quacc each job has its own running directory, keep this in mind when creating workflows.
+    You will often have to use the `copy_files` parameter to copy files from one job to another.
+    You will often want to pass only the previous `dir_name` to copy_file. This interface will
+    then take care of copying the necessary files.
+    3. By default all `pw.x` jobs will use the SSSP pseudopotentials library. This is specified in the
+    `preset` parameter.
```

Comment:
Personally, I would suggest removing this information because it is not related to the calculator specifically. For instance, one could import the `Espresso` class as a standalone utility, but Points 2 and 3 would not be relevant. It is also not particularly accessible to quacc end-users because they are likely to just import the recipes themselves. We should be asking ourselves the question: are the recipes themselves clear enough to use? If not, then we should identify how to clarify _those_ docstrings instead.